docs/USING.rst

```diff
@@ -50,3 +50,70 @@ to hard-disk booting the installed system.
 This is, as the filepath states, merely an example of how you might deploy to
 physical nodes.  Existing deployment systems such as Canonical's MAAS have also
 been run succesfully.
+
+Using CURL Commands
+====================
+
+Included herwith are some examples about interacting with HaaS API using the curl 
+utility.
+
+** Example for registering a Node which use IPMI **
```

Comment:
These still aren't showing up as headings; Maybe you need to get rid of the spaces between the stars and the text?